src/cljs/harja/views/kanavat/urakka/liikenne.cljs

```diff
@@ -211,10 +205,7 @@
         suunta-vaihtoehdot (keys @lt/suunnat-atom)
         uusi-tapahtuma? (not (id-olemassa? (::lt/id valittu-liikennetapahtuma)))
         alukset-atom (atom (zipmap (range) (::lt/alukset valittu-liikennetapahtuma)))
-        toimenpide (::toiminto/toimenpide (first (::lt/toiminnot valittu-liikennetapahtuma)))
-        palvelumuoto (::toiminto/palvelumuoto (first (::lt/toiminnot valittu-liikennetapahtuma)))
-        _ (swap! lt/toimenpide-atom assoc :arvo toimenpide)
-        _ (swap! lt/palvelu-atom assoc :arvo palvelumuoto)]
+        _ (lt/paivita-suunnat-ja-toimenpide! valittu-liikennetapahtuma)]
```

Comment:
Käytetäänkö tässä funktiossa muokattua atomin arvoa tässä funktiossa?
Mietin nimittäin, että voisko tämän atomin muokkauksen tehdä ennenkuin tämä lomake renderöidään?

Reply:
Voi tehdä. Laitoin ennen renderöintiä.

https://github.com/finnishtransportagency/harja/pull/3100/commits/302693c0f4bf08634344e435d7750b24c483a655#diff-1ff2a16dcde2d5da815178764f1e30cddf44ecf32838a7d3c19dc3754f201004R618